packages-next/keystone/src/lib/context/createContext.ts

```diff
@@ -18,32 +18,35 @@ export function makeCreateContext({
   graphQLSchema,
   internalSchema,
   keystone,
-  imagesConfig,
-  filesConfig,
+  config,
+  prismaClient,
+  lists,
 }: {
   graphQLSchema: GraphQLSchema;
   internalSchema: GraphQLSchema;
   keystone: BaseKeystone;
-  imagesConfig: ImagesConfig | undefined;
-  filesConfig: FilesConfig | undefined;
+  config: KeystoneConfig;
+  prismaClient: any;
+  lists: Record<string, { listKey: string; itemQueryName: string; listQueryName: string }>;
```

Comment:
Could you make this `Record<string, GqlNames>` since that's the only thing it's used for?

Reply:
Do you mean:
```
  lists: Record<string, Parameters<typeof getGqlNames>>[0];
```

Reply:
I mean `Record<string, ReturnType<typeof getGqlNames>>` (though don't actually use ReturnType because we export the type that it returns)